src/KIM/kim_init.cpp

```diff
@@ -68,6 +68,7 @@
 #include "modify.h"
 #include "universe.h"
 #include "variable.h"
+#include "fmt/format.h"
```

Comment:
no need to import `fmt/format.h` here since the `KimInit` class is derived from `Pointers`. Any class derived from `Pointers` can assumed that `lmptype.h`, `mpi.h`, `cstddef`, `cstdio`, `string`, `utils.h` and `fmt/format.h` are already included through `pointers.h`.

Reply:
@akohlmey I updated the include statements